VK_App/VK_App/ViewControllers/SecondBarVC/NewsImageCell.swift

```diff
@@ -0,0 +1,18 @@
+// NewsImageCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячейка фото поста
+final class NewsImageCell: UITableViewCell, NewsConfigurable {
+    // MARK: Private IBOutlets
+
+    @IBOutlet private var postImageView: UIImageView!
+
+    // MARK: Public Methods
+
+    func configure(news: NewsFeed, network: NetworkService) {
```

Comment:
```suggestion
    func configure(news: NewsFeed) {
```
так же в сервисе фото запрос